tests/test_hallucination_evaluation.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from deepeval.test_case import LLMTestCase
+from deepeval.metrics.vectara_hallucination_evaluation import (
+    HallucinationEvaluationMetric,
+)
+from deepeval.evaluator import assert_test
+
+
+def test_hallucination_evaluation_metric():
+    metric = HallucinationEvaluationMetric(minimum_score=0.5)
+    test_case = LLMTestCase(
+        input="placeholder",
+        actual_output="A blond drinking water in public.",
+        context=["A man with blond-hair, and a brown shirt drinking out of a public water fountain."],
+    )
+    assert_test(test_case, [metric])
```

Comment:
Two things here:

1. Other than this test, please do provide some tests inside the scorer too `tests/test_scoring.py`.
2. And also, put some couple more tests, (like output type checks, output correctness etc)